src/ucp/api/ucp.h

```diff
@@ -350,7 +350,8 @@ enum ucp_worker_attr_field {
  * present. It is used to enable backward compatibility support.
  */
 enum ucp_listener_attr_field {
-    UCP_LISTENER_ATTR_FIELD_PORT   = UCS_BIT(0) /**< Port used for listening */
+    UCP_LISTENER_ATTR_FIELD_PORT           = UCS_BIT(0), /**< Port used for listening */
+    UCP_LISTENER_ATTR_FIELD_NUM_TRANSPORTS = UCS_BIT(1)  /**< Number of listening transports */
```

Comment:
I think this should be defined the same as UCX_TLS or even reuse this until ucp_listener is created on ifaces